src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -37,7 +37,7 @@ export default class DataTable extends Vue {
   selectedRows: DataTableRow[] = [];
   slicedData: DataTableRow[] = [];
   sortable = false;
-  sortedData?: DataTableRow[];
```

Comment:
If you decorate this property with `@State`, view will re-render when this property is updated.

Reply:
sortedData?: DataTableRow[] = []; <-- this is already reactive

Having an initial value and not having an underscore prefix (_property) change of property value is reactive